### PR TITLE
Feature. Make simple Libarary

### PR DESCRIPTION
**Pull request by TracyMuso**
- We first set up the project using medium-fidelity wireframe in html.
- We then made a function to add books in javascript.
- We then proceeded to make a function that removes books.
- After that we made a function that stores all the books data in the local storage.
- Lastly we debugged checked for linter errors.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      const bookTitle = list.childNodes.value;
+      const remain = bookArr.filter((book) => book.bookTitle !== bookTitle);
+      localStorage.setItem('book', JSON.stringify(remain));
+      newDiv.removeChild(list);
+    }
+  });
+};
+removeBooks();
+// Function to get books from Local Storage
+const getDataFromStorage = () => {
```
Comment (muneebulrehman) [inline index.js]: After adding a few books, followed by reloading the page, the books from the local storage are not populated on the page. Consider fixing it.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (title.value === '' && author.value === '') {
+    alert('Please fill in all the fields');
+  } else {
+    newBook(title.value, author.value);
+  }
+});
+// Remove book function
+const removeBooks = () => {
```
Comment (muneebulrehman) [inline index.js]: The remove functionality doesn't work. It deletes the book from the UI but doesn't delete it from the local storage. Consider fixing it.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @elfin-git @kidde60 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Consider resetting the values inside the input fields. It makes it difficult to add a new book since the user has to clear the input fields from the previous data.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**